instructor-notes/solutions/fileexists_download-fastq.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euo pipefail
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# Define study ID
+STUDY_ID="SRP255885"
+
+# Define file names
+FASTQ_R1="SRR11518889_1.fastq.gz"
+FASTQ_R2="SRR11518889_2.fastq.gz"
+FASTQ_URL="ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR115/089/SRR11518889"
+
+# Define and create destination directory for FASTQ files to live in
+FASTQ_DEST="../data/raw/fastq/$STUDY_ID/"
+mkdir -p $FASTQ_DEST 
+
+##### Process the R1 file #####
+
+# If the file doesn't exist, then we need to download it
+if [ ! -e "$FASTQ_DEST/$FASTQ_R1" ]; then
+	
+	# Print an indicator:
+	echo "Obtaining $FASTQ_R1"
+	
+	# Curl the file (using one of several approaches)
+	curl -O $FASTQ_URL/$FASTQ_R1 # this approach preserves the original internet file name
+
+	# Move the file to its destination directory
+	mv $FASTQ_R1 $FASTQ_DEST
+
+fi
```

Comment:
I'm pretty sure the default for VSCode is to use 4 spaces rather than tabs for bash scripts, (at least that is how it is on my teaching account, where I try not to change defaults) 
```suggestion
if [ ! -e "$FASTQ_DEST/$FASTQ_R1" ]; then
	
    # Print an indicator:
    echo "Obtaining $FASTQ_R1"
	
    # Curl the file (using one of several approaches)
    curl -O $FASTQ_URL/$FASTQ_R1 # this approach preserves the original internet file name

    # Move the file to its destination directory
    mv $FASTQ_R1 $FASTQ_DEST

fi
```